ddtrace/ext/tags.go

```diff
@@ -63,6 +63,27 @@ const (
 	// ServiceName defines the Service name for this Span.
 	ServiceName = "service.name"
 
+	// UserEmail defines the user e-mail span tag
+	UserEmail = "usr.email"
+
+	// UserID defines the user identifier span tag
+	UserID = "usr.id"
+
+	// UserName defines the username span tag
+	UserName = "usr.name"
+
+	// UserRole defines the user role span tag
+	UserRole = "usr.role"
+
+	// UserSessionID defines the user session identifier span tag
+	UserSessionID = "usr.session_id"
+
+	// UserScope defines the user scope span tag
+	UserScope = "usr.scope"
+
+	// PropagatedUserID defines the propagated user identifier, if user id propagation is enabled.
+	PropagatedUserID = "_dd.p.usr.id"
```

Comment:
This is an internal tag. Does the user ever need to set it? If no, it shouldn't be part of the public API. Put it [here](https://github.com/DataDog/dd-trace-go/blob/ef6ccf83444d78a3f276aebd6cdb42407276d531/ddtrace/tracer/span.go#L571) instead.

Reply:
Thanks!